### PR TITLE
SP-86 GetLastTransaction is associating current POS_REF_ID with previously performed transaction

### PR DESCRIPTION
**Pull request by metinavci**

### CODE REVIEW

File: client/src/main/java/com/assemblypayments/spi/Spi.java [hunk 1]
```diff
@@ -1,8 +1,7 @@
 package com.assemblypayments.spi;
 
 import com.assemblypayments.spi.model.*;
-import com.assemblypayments.spi.model.DeviceAddressStatus;
-import com.assemblypayments.spi.service.*;
+import com.assemblypayments.spi.service.DeviceService;
```
Comment (RaymondK-AP) [inline client/src/main/java/com/assemblypayments/spi/Spi.java]: what's the purpose of this change?

Comment (metinavci) [reply client/src/main/java/com/assemblypayments/spi/Spi.java]: The IDE does this automatically.
